cmd/mongodb/main.go

```diff
@@ -0,0 +1,140 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+	"os/signal"
+	"syscall"
+
+	kitprometheus "github.com/go-kit/kit/metrics/prometheus"
+	"github.com/mainflux/mainflux"
+	log "github.com/mainflux/mainflux/logger"
+	"github.com/mainflux/mainflux/writers"
+	mongodb "github.com/mainflux/mainflux/writers/mongodb"
+	nats "github.com/nats-io/go-nats"
+	stdprometheus "github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	name                = "mongodb-writer"
+	defNatsURL          = nats.DefaultURL
+	defPort             = "8180"
+	defDBName           = "mainflux"
+	defDBCollection     = "mainflux"
+	defDBHost           = "localhost"
+	defDBPort           = "8087"
+	defDBUser           = "mainflux"
+	defDBPass           = "mainflux"
+	defDBConnectTimeout = 5000
+	defDBSocketTimeout  = 5000
+
+	envNatsURL          = "MF_NATS_URL"
+	envPort             = "MF_MONGODB_WRITER_PORT"
```

Comment:
Let's drop `DB` from these variables, e.g. `MF_MONGODB_WRITER_PORT` should become `MF_MONGO_WRITER_PORT`, `MF_MONGODB_DB_HOST` should become `MF_MONGO_WRITER_DB_HOST`. Please propagate these changes to InfluxDB writer (@dusanb94 might help here).